data-access-layer/bosh/BoshDirectorClient.js

```diff
@@ -934,20 +934,21 @@ class BoshDirectorClient extends HttpClient {
         params: {
           user_regex: `^${tempUser}`
         }
-      }
-    }, 200, deploymentName);
+      },
+      json: true
+    }, 302, deploymentName);
   }
 
   async runSsh(deploymentName, jobName, instanceId, command) { //jshint ignore: line
     const cryptoManager = new EncryptionManager();
-    const tempUser = `service-fabrik-user-${uuid.v4()}`;
+    const tempUser = `sf-${Math.random().toString(36).substring(2, 15)}`;
```

Comment:
@jayeshjk : Can you add a short comment here on the need for substring with this character size. As I told last week, this is intentionally done to avoid the max username length restriction of "useradd" command on Ubuntu

Reply:
Done. Will push along with other comments.